scripts/dev-test-rc.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
```

Comment:
do we use this file or is this a left over? it looks like a copy of download-rc.sh